metabase/metabase
# Extract VisibilityToggler component from NativeQueryEditor

One more step to solve #18536

Here we simplify `NativeQueryEditor` a bit more.

This should not only make the feature easier to understand, but also bring to light how to excise global Ace so we can replace it with react-ace.

## How to Test

1. Ask a question
2. Native query
3. In the editor, type some query like `select * from PRODUCTS;`
4. Select, then right-click the text you typed.

This toggler should function just as before.

### What we are extracting

![image](https://user-images.githubusercontent.com/380816/138521244-8e63e3db-aeec-4351-9b81-3140b1c72734.png)
